### PR TITLE
Issue 756: Deprecate `http_status`

### PR DESCRIPTION
**Pull request by Aniak5**
#### Description of changes:

- Update redundant http_status to be deprecated 
- Alter descriptions of `latency` and http response status

image

image

### CODE REVIEW

**Review by mr-splunk — DISMISSED** [review]
LGTM

**Review by jp-harvey — DISMISSED** [review]
💯

File: dictionary.json [hunk 1]
```diff
@@ -1530,6 +1530,10 @@
       "type": "http_response"
     },
     "http_status": {
+      "@deprecated": {
+        "message": "Use the <code> http_response.status </code> attribute instead.",
```
Comment (rroupski) [inline dictionary.json:1534]: The http_response has two attributes related to the HTTP response status: `code` and `status`. In this case, the message should recommend the `code`, which has the same type as the deprecated `http_status` (integer):

"message": "Use the <code>http_response.code</code> attribute instead."

**Review by rroupski — CHANGES_REQUESTED** [review]
Similar to the `http_response.status` description, you should update the `http_response.code` description as well. For example:

The Hypertext Transfer Protocol (HTTP) status [code](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml) returned from the web server to the client. For example, <code>200</code>.

File: dictionary.json (hunk 1 above)
Comment (Aniak5) [reply dictionary.json:1534]: Good catch!

**Review by pagbabian-splunk — APPROVED** [review]

**Review by mr-splunk — APPROVED** [review]

**Review by rroupski — APPROVED** [review]

**Pull request merged**